dev_guide/templates.adoc

```diff
@@ -97,15 +150,15 @@ by piping both commands:
 $ oc process -f <filename.json> | oc create -f -
 ----
 
-You can override any link:../dev_guide/templates.html#parameters[parameters]
-defined in the JSON file by adding the `-v` option followed by a
+You can override any link:../dev_guide/templates.html#parameters[parameter]
+values defined in the JSON file by adding the `-v` option followed by a
```

Comment:
I don't like that. I think it is more confusing for users to see JSON/YAML than just JSON , because:

1) all our example application has JSON template
2) all examples in origin repo is JSON
3) all examples in documentation are JSON
4) we mention at 90% of places in docs..

So I would rather have something like "Supported Formats" section somewhere else than here explain that you can use JSON or YAML and we support both formats.